dts/bindings/sensor/st,lis2dh12-i2c.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2018 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
You may want to change copyright here (date and author)

Reply:
It's a straight copy of the existing `lis2dh-i2c.yaml` changing only the product name, so I didn't think it was worth it.  A resolution to #19904 may let us get rid of it anyway.

Reply:
Though since you have this as a change-requested: do you feel this has to be done?

Reply:
I do not think it is strictly needed. But since it is an easy change...
Otherwise I will approve in this way only. It is not a big issue in fact.